core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,70 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A plugin that can efficiently export data from Oracle to be used in Hydrator pipelines.
```

Comment:
its not only useful for Hydrator pipelines, right? It should be a general-purpose export to HDFS?

Perhaps better to say "A Hydrator Action plugin to efficiently export data from Oracle to HDFS or local file system. The plugin uses Oracle's command line tools to export data. The data exported from this tool can then be used in Hydrator pipelines."